cmd/main.go

```diff
@@ -0,0 +1,93 @@
+package cmd
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"os"
+	"time"
+
+	client "github.com/astriaorg/go-sequencer-client/client"
+)
+
+func Execute() {
```

Comment:
rename this to `main()` then you can execute this directly

Reply:
It builds, but I get a `permission denied` error when I try to run the binary.

Reply:
Disregard. I had to update the package name in cmd/main.go!